util/kube/ctl.go

```diff
@@ -78,7 +78,7 @@ func filterAPIResources(config *rest.Config, resourceFilter ResourceFilter, filt
 		}
 		for _, apiResource := range apiResourcesList.APIResources {
 
-			if resourceFilter.IsExcludedResource(gv.Group, apiResource.Kind, config.Host) {
+			if resourceFilter.IsExcludedResource(gv.Group, apiResource.Kind, config.Host, map[string]string{}) {
```

Comment:
I specify empty label in this line.

This function `filterAPIResources` is used in `GetAPIResources`: https://github.com/argoproj/argo-cd/blob/42d572306d252b3ae4b21d3d960fc9d1355c6d5b/util/kube/ctl.go#L125
This function enumerates Kubernetes API resources calling `ServerPreferredResources`, and apply filters. But I can't specify labels in this line, because it takes only resource types and it does not handle individual resources, so it does not contain labels.

I think that there is no problem with empty label. Because this function is used to cache cluster info in: https://github.com/argoproj/argo-cd/blob/8ad928330f26b3eea523626d306306e4929aaa7a/controller/cache/cluster.go#L234 

And I confirmed that it works fine in this state.